docs: Add about defragmentation of PAT (#2146)
groonga/groonga: doc/source/reference/commands/defrag.rst

defragは、対象となるオブジェクト(データベースか可変長サイズカラム)を指定し、オブジェクトのフラグメンテーショ
ンを解消します。

.. versionadded:: 14.1.1

>     Supported patricia trie table. Defragmentation for patrician trie table resolves key fragmentation.

Reviewer: ```suggestion
    You can resolve key fragmentation of a patricia trie table.
```
